geonode/base/api/tests.py

```diff
@@ -325,7 +326,15 @@ def test_base_resources(self):
         self.assertTrue(self.client.login(username='norman', password='norman'))
         response = self.client.get(f"{url}/{resource.id}/", format='json')
         self.assertFalse('change_resourcebase' in list(response.data['resource']['perms']))
-
+        # Check executions are returned when deffered
+        # all resources
+        response = self.client.get(f'{url}?include[]=executions', format='json')
+        self.assertEqual(response.status_code, 200)
+        self.assertIsNotNone(response.data['resources'][0].get('executions'))
+        # specific resource
+        response = self.client.get(f'{url}/{resource.id}?include[]=executions', format='json')
+        self.assertEqual(response.status_code, 200)
+        self.assertIsNotNone(response.data['resource'].get('executions'))
```

Comment:
Can you extend this test in order to check also that the JSON structure of the `executions` element is valid?